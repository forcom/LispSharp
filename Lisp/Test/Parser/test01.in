(define (factorial x)
  (cond ((<= x 1) 1)
	(else (* x (factorial (- x 1))))))

(factorial 5)