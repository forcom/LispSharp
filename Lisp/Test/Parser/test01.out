((define (factorial x) (cond ((<= x 1) 1) (else (* x (factorial (- x 1)))))) (factorial 5))

define : [Lisp.Symbol]
factorial : [Lisp.Symbol]
x : [Lisp.Symbol]
cond : [Lisp.Symbol]
<= : [Lisp.Symbol]
x : [Lisp.Symbol]
1 : [System.Int64]
1 : [System.Int64]
else : [Lisp.Symbol]
* : [Lisp.Symbol]
x : [Lisp.Symbol]
factorial : [Lisp.Symbol]
- : [Lisp.Symbol]
x : [Lisp.Symbol]
1 : [System.Int64]
factorial : [Lisp.Symbol]
5 : [System.Int64]